From 65735c2955876d427379944f6348423bc2b437a6 Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Wed, 6 Dec 2017 11:31:48 +0200
Subject: [PATCH 2997/5242] MLK-16986-5: arm64: dts: fsl-imx8mq.dtsi: Define
 max-res of lcdif node to 720p

commit  ef4148dc62294288a030851e7a007ea48e74b547 from
https://source.codeaurora.org/external/imx/linux-imx.git

Limit the maximum resolution supported by LCDIF to 1280x720, for better
performance when used with DCSS in dual-display mode.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Reviewed-by: Laurentiu Palcu <laurentiu.palcu@nxp.com
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index e9d13f8..36ef4a0 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -594,6 +594,7 @@
 		assigned-clock-parents = <&clk IMX8MQ_VIDEO_PLL1_OUT>;
 		assigned-clock-rate = <594000000>;
 		interrupts = <GIC_SPI 5 IRQ_TYPE_LEVEL_HIGH>;
+		max-res = <1280>, <720>;
 		status = "disabled";
 	};
 
-- 
1.7.9.5

